shelley/chain-and-ledger/executable-spec/test/Examples.hs

```diff
@@ -348,7 +353,7 @@ dariaStake = KeyPair vk sk
 dariaAddr :: Addr
 dariaAddr = mkAddr (dariaPay, dariaStake)
 
--- | Example 1 - apply CHAIN transition to an empty block
+-- * Example 1 - apply CHAIN transition to an empty block
```

Comment:
is this the format for an itemized list

Reply:
I think you have to do
```
-- |
-- * first item
```
but I wasn't able to quickly get the haddocks to build in the test directory...

Reply:
It is format for making a section, since multiple functions below all belong to first example.
https://www.haskell.org/haddock/doc/html/ch03s04.html

Since we also comment on these functions, it is a way for a comment to apply to whole set.

Reply:
awesome!